### PR TITLE
Wave2 619 margins and rounded corners

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module [hunk 1]
```diff
@@ -357,35 +357,36 @@ function ucgy_core_layout_preprocess(&$variables, $hook)
     // Let's make sure this display hasn't already been rendered
 //    if (!isset($variables['display']->ucgy_processed) || !$variables['display']->ucgy_processed) {
     $content = &$variables['display']->content;
+    $module_name = basename(__FILE__, '.module');
+   // $test = $variables['content']['#fieldable_panels_pane']->bundle;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module]: Please remove commented-out code.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module [hunk 2]
```diff
@@ -357,35 +357,36 @@ function ucgy_core_layout_preprocess(&$variables, $hook)
...
-                                        }
-                                        break;
-                                    case "rounded-top":
-                                        // need to set style above and below
-                                        if ($idx > 0 && !isset($content[$idx-1]->css['css_class'])) {
-                                            $content[$page_panes_keys[$idx-1]]->css['css_class'] = "rounded-brick-below container-fluid";
-                                        }
-                                        break;
-                                }*/
+                /*
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module]: Ditto here!

**Comment by nindusaini** [comment]
Thank you Dan for the review, I took out all the commented out code!

**Comment by dpfitzsi** [comment]
Hi Navinder... this one is in conflict. Can you get these resolved?

**Pull request merged**